sympy/series/series.py

```diff
@@ -4,9 +4,55 @@
 
 
 def series(expr, x=None, x0=0, n=6, dir="+"):
-    """Series expansion of expr around point `x = x0`.
+    """Returns
+       =======
+       Series expansion of expr around point `x = x0`.
 
-    See the docstring of Expr.series() for complete details of this wrapper.
+       Parameters
+       ==========
+
+       expr : Expression
+              The expression whose series is to be expanded.
+
+       x    : Symbol
+              It is the variable of the expression to be calculated.
+
+       x0   : Value
+              The value around which ``x`` is calculated. Can be ``oo`` or
+              ``-oo``.
+
+       n    : Value
```

Comment:
```suggestion
       n    : Integer
```

Reply:
for n ,i think changing to integer will be no problem.

Reply:
Infinity object can't be a number.

Reply:
Yeah, but `oo` doesn't work. It gives a TypeError,
And anything between `-oo` and `oo` can be a number.

A third opinion fro this might be a good idea.